websites/F/Fandom/presence.ts

```diff
@@ -1,356 +1,373 @@
-const presence = new Presence({
-  clientId: "644400074008297512"
-});
+if (
+  !(
+    (
+      document.location.pathname.includes("/wiki/") &&
+      !document.querySelector("#globalNavigation")
+    )
+    // Only run on Fandom wikis.
+  )
+)
+  ((): void => {
+    const presence = new Presence({
+      clientId: "644400074008297512"
+    });
 
-let currentURL = new URL(document.location.href),
-  currentPath = currentURL.pathname.replace(/^\/|\/$/g, "").split("/");
-const browsingStamp = Math.floor(Date.now() / 1000);
-let presenceData: PresenceData = {
-  details: "Viewing an unsupported page",
-  largeImageKey: "lg",
-  startTimestamp: browsingStamp
-};
-const updateCallback = {
-    _function: null as () => void,
-    get function(): () => void {
-      return this._function;
-    },
-    set function(parameter) {
-      this._function = parameter;
-    },
-    get present(): boolean {
-      return this._function !== null;
-    }
-  },
-  /**
-   * Initialize/reset presenceData.
-   */
-  resetData = (
-    defaultData: PresenceData = {
+    let currentURL = new URL(document.location.href),
+      currentPath = currentURL.pathname.replace(/^\/|\/$/g, "").split("/");
+    const browsingStamp = Math.floor(Date.now() / 1000);
+    let presenceData: PresenceData = {
```

Comment:
```suggestion
    const browsingStamp = Math.floor(Date.now() / 1000);
    let currentURL = new URL(document.location.href),
      currentPath = currentURL.pathname.replace(/^\/|\/$/g, "").split("/"),
      presenceData: PresenceData = {
```

Reply:
Yep. That's going to be an base update. Expect updates for all 19 prences.